src/main/java/frc/trigon/robot/misc/objectdetectioncamera/ObjectDetectionCamera.java

```diff
@@ -21,36 +24,53 @@ public void periodic() {
         Logger.processInputs(hostname, objectDetectionCameraInputs);
     }
 
+    public void trackObject() {
+        if (hasTargets() && !wasVisible) {
```

Comment:
1. Add a jdoc
2. Is this how it was in the offseason code? I remember we did something else.

Reply:
I copied over the code from the offseason and modified that...